literature/durand_1992_insitu_1977/durand_1992_insitu_1977_p1_f1a.yaml

```diff
@@ -0,0 +1,66 @@
+curation:
+  process:
+  - date: 2022-02-18
+    name: Tobias Schmider
+    orcid: https://orcid.org/0000-0002-2911-2925
+    role: curator
+  version: 1
+electrochemical system:
+  electrodes:
+    configuration: three
+    reference electrode:
+      type: RHE
+    working electrode:
+      crystallographic orientation: '110'
+      geometric electrolyte contact area:
+        unit: cm-2
+      material: Pt
+      preparation procedure: Annealed in an H2-O2 flame.
+  electrolyte:
+    components:
+    - name: water
+      proportion:
+        unit: volume percent
+        value: 100
+      type: solvent
+    - concentration:
+        unit: mol / l
+        value: 0.001
+      name: HClO4
+      type: acid
+    - concentration:
+        unit: mol / l
+        value: 1
+      name: NaClO4
+      type: salt
+    - concentration:
+        unit: mol / l
+        value: 0.0001
+      name: Cu
+      purity:
+        grade: 5N
+      type: salt
+      comment: Cu-ion source not provied.
```

Comment:
This is a special case. In the article, the source of the Cu ions is not given.
I am not sure how to handle this properly but would go and call this a salt anyway. Maybe we can assume that it is CuO and leave a comment. Another option is to find out if the authors indicated this value in other publications.
@linuxrider   any other suggestions?

Reply:
Sorry for the late reply. Do we need to obey electroneutrality in the components? If not we could have a type `ion`?
In ref 12, which is mentioned in the experimental section, recrystallized sulfate is used, maybe here the perchlorate salt is used. I would go with the new type route.